src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy

```diff
@@ -16,5 +16,6 @@ class InspectContainerResponseHandler implements ResponseHandler<String, Object>
         logger.quiet "VolumesFrom : $container.hostConfig.volumesFrom"
         String exposedPorts = container.config?.@exposedPorts ? container.config.exposedPorts : '[]'
         logger.quiet "ExposedPorts : $exposedPorts"
+        logger.quiet "LogConfig : $container.hostConfig.logConfig.type.type"
```

Comment:
The first 'type' returns an Enum while the second 'type' returns the string representation of that given enum.

https://github.com/docker-java/docker-java/blob/master/src/main/java/com/github/dockerjava/api/model/LogConfig.java